### PR TITLE
 Pass unit tests on all screen resolutions

### PR DESCRIPTION
**Pull request by dfed**
Plus some project cleanup while we were here.

cc @martinmroz @nickentin

### CODE REVIEW

File: Tests/SubviewDistributionTests.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ class SubviewDistributionTests: XCTestCase {
         let pixelRounder = PixelRounder(for: view)
         // Assert that our spacing is correct above.
         XCTAssertEqual((a.bottom |--| b.top).height, relative1Space, accuracy: pixelRounder.pixelAccuracy)
-        XCTAssertEqual((b.bottom |--| c.top).height, 2 * relative1Space, accuracy: pixelRounder.pixelAccuracy)
+        XCTAssertEqual((b.bottom |--| c.top).height / 2.0, relative1Space, accuracy: pixelRounder.pixelAccuracy)
```
Comment (dfed) [inline Tests/SubviewDistributionTests.swift:50]: Previously we were multiplying our rounding delta. Now we're dividing the actual measured space and comparing it to the relative space within a pixel distance.

File: Sources/SubviewDistributor.swift [hunk 2]
```diff
@@ -175,18 +175,18 @@ public struct SubviewDistributor {
...
-            let bottomVerticalInset = (superview.bounds.maxY - distributionRect.maxY)
-            
+            let leftHorizontalOffset = distributionRect.minX
+            let horizontalMidpoint = distributionRect.midX - leftHorizontalOffset
+            let topVerticalOffset = distributionRect.minY
+            let verticalMidpoint = distributionRect.midY - topVerticalOffset
+
             switch direction {
             case .vertical:
-                return superview.top + UIOffset(horizontal: (leftHorizontalInset - rightHorizontalInset) / 2.0,
```
Comment (dfed) [inline Sources/SubviewDistributor.swift:185]: The previous flow here increased the chances of a rounding error by rounding both the `superview`'s midpoint and the `distributionRect`'s midpoint. Now we're just rounding the midpoint on the distributionRect.

File: Relativity.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -430,7 +428,6 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.dfed.RelativityTests;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```
Comment (dfed) [inline Relativity.xcodeproj/project.pbxproj:433]: given that we have no Objective-C, this flag never should have been on anyways.

**Comment by martinmroz** [comment]
LGTM

**Pull request merged**